pkg/hvs/controllers/flavor_controller.go

```diff
@@ -175,11 +184,25 @@ func (fcon *FlavorController) createFlavors(flavorReq dm.FlavorCreateRequest) ([
 			return nil, errors.Wrap(err, "Error while generating a formatted connection string")
 		}
 		defaultLog.Debug("Getting manifest from host...")
+
 		hostManifest, err := fcon.getHostManifest(connectionString)
 		if err != nil {
 			defaultLog.Error("controllers/flavor_controller:CreateFlavors() Error getting host manifest")
 			return nil, errors.Wrap(err, "Error getting host manifest")
 		}
+
+		var flavorTemplates []hvs.FlavorTemplate
+		if !strings.EqualFold(hostManifest.HostInfo.OSName, "VMWARE ESXI") {
```

Comment:
For ESXI flow flavor template will not be applied. Templates only gets applied to linux flavors.

Reply:
This does not appear to negate any requirements in the Event-Log design document.  However, the implication is that customers will not be able to deploy custom validation (aka templates) for ESXI.  I think we should elevate this so that stakeholders are at least aware of the limitation and if needed, add an enhancement to our backlog.